cmd/talisman/talisman.go

```diff
@@ -75,4 +82,7 @@ func init() {
 	flag.StringVar(&newPXTag, "pxtag", "", "(optional) New Portworx tag to use for the upgrade")
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "(optional) Absolute path of the kubeconfig file")
 	flag.StringVar(&dockerRegistrySecret, "dockerregsecret", "", "(optional) Kubernetes Secret to pull docker images from a private registry")
+	flag.StringVar(&sharedAppsScaleDown, "scaledownsharedapps", string(px.SharedAppsScaleDownAuto),
+		fmt.Sprintf("(optional) instructs scale down behavior of px shared apps. Supported values: %s, %s, %s",
```

Comment:
May want more text here explaining. This is confusing for those not used to px. Also, may want to spell out Portworx instead of just using `px`.

Reply:
In incremental, updated text to 

```
  -scaledownsharedapps string
    	(optional) instructs scale down behavior of Portworx shared apps. Supported values:
	auto: During the upgrade process, Portworx shared applications will be scaled down to 0 replicas if 1.2 to 1.3 version upgrade is detected.
	on: During the upgrade process, Portworx shared applications will be unconditionally scaled down to 0 replicas.
	off: Upgrade process will not scale down Portworx shared applications. (default "auto")
```